recipes/serpenttools/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "serpentTools" %}
+{% set version = "0.10.1" %}
+
+package:
+  # Do need the package name to be lower cased here, even though the
+  # package has that dang upper case T
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: da6fbd0db6a55538720a1150cbd146c94dd5049f75b8bd1f9eb83c30c213360b
+
+build:
```

Comment:
Are you sure that this cannot be noarch?

https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Reply:
Definitely can be `noarch`. Added via `grayskull`